l10nru.properties

```diff
@@ -10,14 +10,14 @@ defaults.mavenArtifactId=sonar-l10n-ru-plugin
 10.8.description=Support SonarQube 10.8 (SonarQube Core 24.12)
 10.8.date=2025-01-07
 10.8.sqs=[10.8,LATEST]
-10.8.sqcb=[24.12,LATEST]
+10.8.sqcb=[25.1,LATEST]
```

Comment:
FYI @theshadowco. Compatibility of public versions can't overlap, so this has been bumped up.